LICENSE.old

```diff
@@ -1,21 +0,0 @@
-The MIT License (MIT)
```

Comment:
This file can't be removed without explicit authorization of the original author.